packages/vhd-lib/merge.integ.spec.js

```diff
@@ -28,11 +29,13 @@ afterEach(async () => {
   await pFromCallback(cb => rimraf(tmpDir, cb))
 })
 
-async function createRandomFile(name, sizeMb) {
-  await execa('bash', [
-    '-c',
-    `< /dev/urandom tr -dc "\\t\\n [:alnum:]" | head -c ${sizeMb}M >${name}`,
-  ])
+async function createRandomFile(name, size) {
+  await fromEvent(
+    createReadStream('/dev/urandom', { end: size * 1024 * 1024 - 1 }).pipe(
+      fs.createWriteStream(name)
+    ),
+    'finish'
+  )
```

Comment:
Instead of `fromEvent`, you can use [`readable-stream/pipeline`](https://devdocs.io/node~10_lts/stream#stream_stream_pipeline_streams_callback) (with `fromCallback` or promisified) which is more robust:

```js
await fromCallback(cb => pipeline(input, output, cb))
```

I'm trying to migrate all manual `.pipe`s to it progressively.